core/src/main/java/bio/terra/pearl/core/service/export/JsonExporter.java

```diff
@@ -18,15 +18,19 @@ public JsonExporter(List<ModuleFormatter> moduleFormatters, List<Map<String, Str
         this.objectMapper = objectMapper;
     }
 
-    public void export(OutputStream os) throws IOException {
+    public void export(OutputStream os) {
         PrintWriter printWriter = new PrintWriter(os);
         List<String> columnKeys = getColumnKeys();
         List<String> headerRowValues = getHeaderRow();
         List<String> subHeaderRowValues = getSubHeaderRow();
 
         JsonExport jsonExport = new JsonExport(columnKeys, headerRowValues, subHeaderRowValues, enrolleeMaps);
-        printWriter.println(objectMapper.writeValueAsString(jsonExport));
-        printWriter.flush();
+        try {
+            printWriter.println(objectMapper.writeValueAsString(jsonExport));
+            printWriter.flush();
+        } catch (IOException e) {
+            throw new RuntimeException("Error writing json to stream", e);
```

Comment:
Here, and in ExcelExporter and DataDictionaryExcelExporter can you change the thrown exception to something more specific than RuntimeException, which seems ambiguous from an error response perspective? 
I assume we want to return a 500, or perhaps a 503.

Also, does it make sense to indicate in the message that retrying the export may work?

Reply:
It's anyone's guess whether a retry will work, could be due to server conditions, or could be due to some really unexpected data properties.  I don't necessary want to encourage a retry, for example, the reason the writing failed was the server ran out of RAM after loading a large study.